R/pkg/R/DataFrame.R

```diff
@@ -1324,12 +1312,16 @@ setMethod("selectExpr",
 #' path <- "path/to/file.json"
 #' df <- jsonFile(sqlContext, path)
 #' newDF <- withColumn(df, "newCol", df$col1 * 5)
+#' # Replace an existing column
+#' newDF2 <- withColumn(newDF, "newCol", newDF$col1)
```

Comment:
I'm not 100% about the replace existing column behavior - I thought it was intentional that we support multiple columns with the same name before?